shared/cfg/guest-os/Linux/Fedora/24.x86_64.cfg

```diff
@@ -0,0 +1,24 @@
+- 24.x86_64:
+    image_name = images/f24-64
+    vm_arch_name = x86_64
+    os_variant = fedora24
+    no unattended_install..floppy_ks
+    unattended_install, svirt_install:
+        kernel_params = 'inst.repo=cdrom:/dev/disk/by-label/Fedora-S-24-x86_64'
```

Comment:
I've added the Trello card link, and I'll put a BZ link there (in trello) when I confirm (I want to double check) it's a Fedora bug.